doc/whats_new.rst

```diff
@@ -92,6 +92,8 @@ API
 
 - Target cell types and their connections are created for each drive according to the synaptic weight and delay dictionaries assigned in `Network.add_xxx_drive()`, by `Ryan Thorpe`_ in `#369 <https://github.com/jonescompneurolab/hnn-core/pull/369>`_
 
+- Cell objects can no longer be accessed from :class:`~hnn_core.Network` as the :attr:`~hnn_core.Network.cells` attribute has been removed, by `Ryan Thorpe`_ in `#436 <https://github.com/jonescompneurolab/hnn-core/pull/436>`_
```

Comment:
just merged #430 so perhaps adjust to use the new substitution

Reply:
fixed in 5325317